src/gravity/Artwork.ts

```diff
@@ -1,103 +1,230 @@
-import {
-  Boolean,
-  Number,
-  String,
-  Array,
-  Record,
-  Static,
-  Undefined,
-  Null,
-} from "runtypes";
-import { EmbeddedArtist } from "./Artist";
-import { EditionSet } from "./EditionSet";
-import { Image } from "./Image";
-import { CulturalMaker } from "./CulturalMaker";
-import { Partner } from "./Partner";
```

Comment:
Ultimately I think composing types from other places is too error prone. Each type should ultimately represent an individual API response.